### ISSUE TITLE
Amend Project Estimation and plan

### ISSUE THREAD

**Issue by joash2808** [jevancjk/ICT2201-P1-4#53]

**Comment by joash2808** [comment]
The number of use case descriptions must match use case diagram
The work time we obtained via estimation is not doable and realistic. We need to talk to client about reducing the requirements in order to make the hours more doable.
In the milestone report, write about how the hours in not doable, why its not doable and how we need to talk to the client about cutting down on requirements.

**Comment by joash2808** [comment]
Gantt chart
- Need to consider SDLC
- Gantt chart needs show the whole duration of the project and what we have planned. Changes can be made later but for now just plan everything.
- Need to make a description of our project management strategy.
- We are doing hybrid of waterfall and agile.

**Comment by YapJiaHao** [comment]
The new UCP would be 117 * 1.095 * 1.265 = 162 (rounded off)
Estimation would be 15 * 162 to 30 * 162
2430 - 4860 hours.
Among 4 teammates, which would be 607.5 hours per person.
This value is reasonable when considering that it takes an average of 600 total hours in developing a simple web app, based on [https://spdload.com/blog/average-time-to-create-a-website/](url).
But given that we only have 12 weeks worth of time, some functionalities would have to be reduced in order to meet the deadline.

**Comment by chewweije** [comment]
Agree with above, final estimation need to be reduced. But have to request reduced requirements from client first.

**Comment by jevancjk** [comment]
Agreed, best to get in touch with the client again on the reduced functionalities.

**Comment by YapJiaHao** [comment]
The new UCP would be 107 * 1.095 * 1.265 = 148
Estimation would be 15 *  148 to 30 * 148
2220 - 4440 hours.
In a best case scenario, it would take 555 hours per person on the project.
